docs/semgrep-secrets/generic-secrets.md

```diff
@@ -0,0 +1,60 @@
+---
+slug: generic-secrets
+title: Scan for generic secrets
+hide_title: true
+description: Use Semgrep to identify generic secrets in your code.
+tags:
+ - Semgrep Assistant
+ - Semgrep Secrets
+---
+
+import PL from '@site/src/components/Placeholder';
+
+# Generic secrets (beta)
+
+:::info
+Generic secrets is in beta. Please contact [Support](/support) to request access to this feature.
+:::
+
+Generic secrets detection scans your code for the inadvertent inclusion of credentials, such as API keys, passwords, and access tokens using rules. It then analyzes the results to eliminate false positives, so you only see high-signal results likely to be true positives.
+
+## Prerequisites
+
+To scan your code for generic secrets, you must have the following:
+
+- [Repositories hosted by GitHub.com onboarded to Semgrep using Semgrep Managed Scans](/deployment/managed-scanning/github)
+- Access to [Semgrep Secrets](/semgrep-secrets/getting-started)
+- [Semgrep Assistant](/semgrep-assistant/getting-started) enabled
+- Semgrep CLI version `1.86.0` or higher
+
+Generic secrets does *not* work with local scans initiated by running the `semgrep ci` command.
```

Comment:
I understand that the Semgrep CLI version requirement isn't for local machines (it's for the docker image that runs Semgrep) but on first read the two seem to contradict each other. Is there a way to make it clear what the CLI version is for?